### ISSUE TITLE
Alt Text

### ISSUE THREAD

**Issue by daniel-coleman-1** [ONRR/onrr.gov-site#1743]
Alt text is contained in this file:
https://doimspp.sharepoint.com/:w:/s/NRRD/EanXkHsMmaRPlt1lIsNq_TQB0Lh7nvYoptrOcPtpf1OZZQ?e=DLlvs1

@cthomasONRR I'm seeing a ton of overlap, but not in the same order.

**Comment by daniel-coleman-1** [comment]
@cthomasONRR I noticed the dev selection was alphabetized so I alphabetized the local one as well, they're the same.  I'm going to start plugging them into the CMS.  There are some areas where I may ask for clarification around punctuation, like here:
![Screenshot 2022-03-22 162945](https://user-images.githubusercontent.com/83892362/159586970-fe0c546d-c872-4a1c-9ad1-1559bd7538ee.png)
.

**Comment by daniel-coleman-1** [comment]
@cthomasONRR I'm going to wait on plugging these into the CMS until you can take a look, I think we need to re-add all of the commas and may want to re-write some of the alt text.

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 Can you make a list for @cthomasONRR for any images that are technical that are missing alt text? The punctuation errors I see in the spreadsheets are quotation marks that didn't import correctly, the example you mentioned above should be "Contractual Allocated Fuel".

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 why do we need to re-write alt text? @cthomasONRR has already spend time writing these alt texts with detail before they were deleted as you were working in issue #1717.  You can clean up what Mojo gave you and add any commas and start adding to the CMS. If you come across any technical images that don't have alt texts then Christine can write them. But you should have what you need to complete issue#1717.

**Comment by daniel-coleman-1** [comment]
@Maroyafaied I thought there were some areas we may want to be more descriptive, like:
"2.5.5.1 Image #1" "Flow chart for deciding how to value Indian Gas""" 

but if you think it's good to go, I'll plug them in.

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 Yes you can plug everything to the CMS and also add alt texts to images that are not technical. Again, you would need to clean up the quotation marks and add necessary commas.  While you are identifying technical images that alt text have been deleted and need to be re-written, you can also note any images you feel need to be more descriptive to that list.  Since @cthomasONRR has already spend time writing alt texts, I'd like for her to have a clean list of any missing terms and any that you think need to be more descriptive. Also, since the image titles have changed, can you include in that same list what the new image title is for @cthomasONRR to be able to quickly go to the image folder and re-write or add more description.

**Comment by daniel-coleman-1** [comment]
@cthomasONRR only the following 3 require alt text:

Reporting Codes
![Screenshot 2022-03-23 151743](https://user-images.githubusercontent.com/83892362/159797640-104f41a8-c7b1-48a6-9b5c-a734c73e6455.png)

Apop 7
![Screenshot 2022-03-23 151702](https://user-images.githubusercontent.com/83892362/159797655-287b3933-e64a-44e2-abbd-c03ace45aebb.png)

Indian Gas Valuation Flow Chart
![Screenshot 2022-03-23 151908](https://user-images.githubusercontent.com/83892362/159797703-45db404c-46b0-457d-81e2-bdaa432820b2.png)

**Comment by daniel-coleman-1** [comment]
@cthomasONRR I'll have a list for you tomorrow of all of the alt text I'd like to change, accompanied by suggestions to accept/decline.  Should be about 12-15 items.

**Comment by cthomasONRR** [comment]
@daniel-coleman-1 I added alt text for the 2 images above.
